packages/grid/src/styles/column.module.css

```diff
@@ -0,0 +1,688 @@
+:root {
+  --column-gap: var(--column-gap);
+  --column-width: calc((100% - (11 * var(--column-gap))) / 12);
```

Comment:
please keep the naming rules of the design tokens.
`[main purpose][internal value][auxiliary]`
`--grid-column-gap`